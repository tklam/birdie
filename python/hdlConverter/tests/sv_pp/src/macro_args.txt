`define m0 "abc"
`define m1(x) x

`m0(123)
`m0(123, "a"
 , "b", 1++<<<
    >> )
`m1(456)
// `m2(