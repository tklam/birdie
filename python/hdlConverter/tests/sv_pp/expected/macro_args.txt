


"abc"(123)
"abc"(123, "a"
 , "b", 1++<<<
    >> )
456
// `m2(